modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -13,6 +13,263 @@
+* Web: https://github.com/owncloud/web/releases/tag/v8.0.0[Changelog for ownCloud Web 7.1.0]
+
+[discrete]
+=== Federated Cloud Sharing
+
+Federated Cloud Sharing allows users to access and collaborate on files stored on connected external servers, as if they were stored on the user's instance. This creates a fluid and efficient workflow, eliminating the need for multiple file versions or complex file transfer processes. With Federated Cloud Sharing data can reside on its original server, ensuring compliance with regional data sovereignty laws and organizational data policies.
+
+To enable file sharing with a user on a different, connected server, an initial invitation needs to be sent. Once this is completed, files can be shared with that user as usual by entering their username, just like with internal users. The only difference is that users from connected instances will be marked as "Federated Users" for easy identification and management.
+
+*Technical Note:* Federated Cloud Sharing in Infinite Scale is based on the Open Cloud Mesh protocol version 1.1 (OCM 1.1) https://cs3org.github.io/OCM-API/docs.html?branch=v1.1.0&repo=OCM-API&user=cs3org#/paths/~1shares/post[Learn more about OCM 1.1]
+
+Kudos to all members of the CS3-Community who contributed to this project. A special thanks is owed to CERN for their contribution of the Science Mesh App. Thank You!
+
+[discrete]
+=== Password Policy for Sharing Links
+
+The Password Policy for Sharing Links allows administrators to set specific requirements for password strength. This feature ensures that your sharing practices align seamlessly with existing security policies.
+
+The customizable settings for the password are minimum amount of:
+
+* uppercase characters
+
+* lowercase characters
+
+* numbers
+
+* the special characters: "!#\$%&'()*+,-./:;<=>?@[\]^_`{|}~"
+
+* and password length
+
+
+**Banned Passwords List** 
+To further bolster security, this update includes the ability to define a banned password list. This feature prevents users from using overly common or simplistic passwords, like 'Password123!', thereby reducing the risk of unauthorized access.
+
+**Integrated Password Generator**
+
+Accompanying this new password policy is a built-in password generator. It automatically creates passwords that comply with your defined requirements, simplifying the process for users while maintaining a high standard of security.
+
+[discrete]
+=== Filter Chips: Type and last modified
+
+[discrete]
+==== Type Filter Chip
+Easily narrow down your search by file type. Whether you're looking for documents, images, spreadsheets, or any other file types, the Type Filter Chip allows you to quickly filter results to match your specific needs.
+The Type filter groups the following MIME types.
+(The usual file extension is indicated in parentheses after the MIME type for ease of reference. However, it is the MIME type of a file that is decisive, not the file extension.)
+
+* **Document Filter:**
```

Comment:
Little bit surprised to see the mimetypes in the release notes. IMO only something for the changelog. Or maybe put a link to documentation...

Reply:
hm yes, feels really verbose. so lets go with the disclaimer that we index the mimetype and not the file extension